src/components/InMarketAsset/InMarket.tsx

```diff
@@ -45,9 +48,13 @@ const InMarket = () => {
   const [delistContainerHeight, setDelistContainerHeight] = useState<string>('120px')
   const [openModal, setOpenModal] = useState<boolean>(false)
   const [disabled, setDisabled] = useState<boolean>(false)
-  const expired = auction.expireTimestamp - Date.now()
+  const expired = auction?.expireTimestamp - Date.now()
```

Comment:
to resolve auction expiretime is null should be validate that auction data is called